lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -61,6 +61,12 @@ sub _group_result {
         while (my $line = $keys->next) {
             $settings{$line->job_id}->{$line->key} = $line->value;
         }
+        # prefetch comments to count
+        my $c = $self->db->resultset("Comments")->search({job_id => {in => [map { $_->id } $jobs->all]}});
+        my %labels;
```

Comment:
Isn't label comment just in special format?
In [Test.pm:345](https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/WebAPI/Controller/Test.pm#L345) `%labels` include comments too.